portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,24 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Alex Najera</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Alex Najera</h1> 
+      <img class="image1" src="/images/meface.png" />
```

Comment:
Minor thing, it's often better to have more meaningful class names (see style guide for some more info in general, https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming).
In this case, I can see using image1 if we always want the first image on the page to be styled like this, but even better might be something like header-image.